src/video.c

```diff
@@ -1267,8 +1273,10 @@ int video_update(struct video *v, const char *peer)
 	else
 		video_stop_source(v);
 
-	if (dir & SDP_RECVONLY) {
+	if (dir == SDP_RECVONLY)
```

Comment:
@cspiel1 can you have a look if this fits your needs? the natpinhole is only needed if TX thread is not started, right?

Reply:
Right! It seems to be not correct currently. It should be:
```c
	if (dir == SDP_RECVONLY)
		err |= stream_open_natpinhole(v->strm);

	if (dir & SDP_RECVONLY) {
		err |= video_start_display(v, peer);
	}
	else {
		video_stop_display(v);
	}
```